src/utils/utilities.go

```diff
@@ -65,5 +66,9 @@ func MaskCredentialsInUrl(url string) string {
 // Remove invalid characters from the stat name.
 func SanitizeStatName(s string) string {
 	r := strings.NewReplacer(":", "_", "|", "_")
-	return r.Replace(s)
+	ipv4Pattern := `\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}`
```

Comment:
would it be better to make this as const?

Reply:
optimized it a bit by moving the static regex compilation out of the function